internal/envtest/environment.go

```diff
@@ -125,6 +125,12 @@ func New(uncachedObjs ...client.Object) *Environment {
 		err = kerrors.NewAggregate([]error{err, env.Stop()})
 		panic(err)
 	}
+	u, _ := env.AddUser(envtest.User{Name: "debug", Groups: []string{"system:masters"}}, env.Config)
+
+	cfg, _ := u.KubeConfig()
+	fd, _ := os.Create("kubeconfig")
+	fd.Write(cfg)
+	defer fd.Close()
```

Comment:
It seems this is a debug kubeconfig, can you please remove it? It may also fix the linter failure, but you can also run `make lint-fix` to verify.

Reply:
It is!
It shouldn't have been committed.
I will obviously remove it